sdk/appcomplianceautomation/Azure.ResourceManager.AppComplianceAutomation/tests/Tests/ReportTests.cs

```diff
@@ -0,0 +1,70 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading.Tasks;
+using Azure.Core.TestFramework;
+using Azure.ResourceManager.Resources;
+using Azure.ResourceManager.AppComplianceAutomation;
+using NUnit.Framework;
+using Azure.ResourceManager.AppComplianceAutomation.Models;
+using System.Collections.Generic;
+using Azure.Core;
+
+namespace Azure.ResourceManager.AppComplianceAutomation.Tests.Tests
+{
+    public class ReportTests : AppComplianceAutomationManagementTestBase
+    {
+        public ReportTests(bool isAsync) : base(isAsync)
+        {
+        }
+
+        private TestContext testContextInstance;
+
+        public TestContext TestContext
+        {
+            get { return testContextInstance; }
+            set { testContextInstance = value; }
+        }
+
+        [SetUp]
+        public void Init()
+        {
+            InitializeUserTokenClients();
+        }
+
+        [OneTimeTearDown]
+        public void Cleanup()
+        {
+        }
+
+        [TestCase]
+        public async Task ReportCRUD()
+        {
+            string reportName = Recording.GenerateAssetName("sdk-report");
+            ReportResourceCollection reports = new ReportResourceCollection(Client, new ResourceIdentifier("/providers/Microsoft.Resources/tenants/72f988bf-86f1-41af-91ab-2d7cd011db47"));
```

Comment:
Nit: This could work but we should avoid using the internal ctor in tests, can we use following code to get the reports collection?
```csharp
TenantCollection tenantsCollection = Client.GetTenants();
var tenants = await tenantsCollection.GetAllAsync().ToEnumerableAsync();
var tenant = tenants.FirstOrDefault();
ReportResourceCollection reports = tenant.GetReportResources();
```